agenta-web/src/services/evaluations/index.ts

```diff
@@ -238,7 +244,9 @@ export const fetchAllComparisonResults = async (evaluationIds: string[]) => {
             inputs: inputNames
                 .map((name) => ({name, value: data[name]}))
                 .filter((ip) => ip.value !== undefined),
-            correctAnswer: data.correct_answer || "",
+            ...correctAnswers.reduce((acc, curr) => {
+                return {...acc, [`correctAnswer_${curr?.key}`]: data[curr?.key!]}
+            }, {}),
```

Comment:
why do we have to do this 
and why it's in the evaluation and not in the compare